pulp_smash/utils.py

```diff
@@ -9,7 +9,7 @@
 from urllib.parse import urlparse
 
 import requests
-
+from pulp_smash import exceptions
 from pulp_smash import cli
```

Comment:
This can be written as:

    from pulp_smash import cli, exceptions

No need to have separate imports.